### ISSUE TITLE
Cisco FTD Security Intelligence Logs

### ISSUE THREAD

**Issue by cchansk** [splunk/splunk-connect-for-syslog#1442]
I am having an issue with Cisco FTD logs and I'm not quite sure why. The logs end up in sc4s:fallback, but there should be a parser for FTD according to the docs and issue 466 and 571. Not quite sure what I might be doing wrong. Below is a sample of the log, was hoping for some help with getting them parsed.

2022-02-07T22:06:17Z hostname %FTD-6-430002: EventPriority: High, DeviceUUID: 00000000-0000-0000-0000-000000000000, InstanceID: 2, FirstPacketSecond: 2022-02-07T22:06:17Z, ConnectionID: xxxxx, AccessControlRuleAction: Block, AccessControlRuleReason: IP Block, SrcIP: xxx.xxx.xxx.xxx, DstIP: xxx.xxx.xxx.xxx, SrcPort: xxx, DstPort: xxx, Protocol: tcp, IngressInterface: interface_name, EgressInterface: interface_name, ACPolicy: policy_name, Prefilter Policy: Default Prefilter Policy, InitiatorPackets: x, ResponderPackets: x, InitiatorBytes: x, ResponderBytes: x, NAPPolicy: policy_name, SecIntMatchingIP: Destination, IPReputationSICategory: category_name

**Comment by rfaircloth-splunk** [comment]
Hi this is caused by a defect in the Cisco firmware failing to send the syslog pri in the header. Because this is a persistent defect we found a work around published in v2.0.0 today

**Comment by cchansk** [comment]
Thanks... I updated and it does appear to be parsing it with the cisco_ftd_nopri format. I took a look at some Cisco docs and at 6.3 and up, if you turn on EMBLEM mode, then it will send the pri. I've since turned it on and the syslog format switched to rfc3161_isodate. However, it seems to be recognizing it as cisco_asa as the vendor product/sourcetype and asa as the product. Is the correct? I thought it would recognize it as cisco:firepower or cisco:ftd.
